readme.md

```diff
@@ -1,7 +1,7 @@
 # Boilerplate for layout tasks
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_snake/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_snake/report/html_report/)
+- [DEMO LINK](https://RomanTsvindin.github.io/layout_snake/)
```

Comment:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/38065883/167590462-d8e9ea46-95e6-40ca-8094-f5cf18c84762.png">
It should take full width on big screens